tests/bootstrap.php

```diff
@@ -1,90 +1,120 @@
 <?php
+declare(strict_types=1);
 
-use Cake\Core\Plugin;
-// @codingStandardsIgnoreFile
-
-$findRoot = function () {
-    $root = dirname(__DIR__);
-    if (is_dir($root . '/vendor/cakephp/cakephp')) {
-        return $root;
-    }
+/**
+ * Test suite bootstrap.
+ *
+ * This function is used to find the location of CakePHP whether CakePHP
+ * has been installed as a dependency of the plugin, or the plugin is itself
+ * installed as a dependency of an application.
+ */
 
-    $root = dirname(dirname(__DIR__));
-    if (is_dir($root . '/vendor/cakephp/cakephp')) {
-        return $root;
-    }
+use Cake\Cache\Cache;
```

Comment:
Please update this to use the generic bootstrapping code as done for Throttle plugin.